Library/Homebrew/manpages/brew.1.md

```diff
@@ -556,6 +556,14 @@ can take several different forms:
 
     *Note:* Homebrew will only try to print emoji on Lion or newer.
 
+  * HOMEBREW\_NO\_INSECURE\_REDIRECT:
+    If set, Homebrew will not permit redirects from secure download
+    protocols to insecure ones.
+
+    Whist ensuring your downloads are fully secure, this is likely
```

Comment:
Seems to be a British/American lexicon difference thing. I'm happy to change it.